src/body.js

```diff
@@ -204,8 +206,8 @@ function consumeBody() {
 		return Body.Promise.resolve(this.body);
 	}
 
-	// body is buffer
-	if (Object.prototype.toString.call(this.body) === '[object ArrayBuffer]') {
+	// body is ArrayBuffer
+	if (this.body instanceof ArrayBuffer) {
 		return Body.Promise.resolve(Buffer.from(this.body));
 	}
```

Comment:
ArrayBufferViews don't seem to be handled here.

Reply:
It was here initially, but code coverage showed that `ArrayBufferView` does't appear here. When `ArrayBufferView` is sent, only `ArrayBuffer` seems to appear on receiving side. Or am I missing something?

Reply:
It is there for cases like

```js
const req = new Request('about:blank', {
  method: 'POST',
  body: new Uint16Array([32123])
});
const j = await req.json();
```

Reply:
Interesting, but I do not see any explicit use of `Request` in existing tests.

Reply:
I'm just not familiar enough with code base yet, sorry about that. Makes sense to add even more test cases here. As usual, seems so simple on the surface, but quite complex under the hood.